wled00/wled_server.cpp

```diff
@@ -21,7 +21,7 @@ static const char s_accessdenied[]   PROGMEM = "Access Denied";
 static const char _common_js[]       PROGMEM = "/common.js";
 
 //Is this an IP?
-static bool isIp(String str) {
+static bool isIp(const String str) {
```

Comment:
Perhaps even better with `const String &str` as it will avoid copy constructor.

Reply:
thanks, yes that's a good idea.